packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -125,16 +137,20 @@ export class AtomicNumericFacet
 
   public initialize() {
     this.searchStatus = buildSearchStatus(this.bindings.engine);
-    this.manualRanges = this.buildManualRanges();
+    this.initializeFacet();
```

Comment:
Perhaps when "withRanges" is false, I could not create the facet controller, this could make so the component has more condition.